bin/kubeconfig.bash

```diff
@@ -7,37 +7,16 @@ here="$(dirname "$(readlink -f "$0")")"
 source "${here}/common.bash"
 
 usage() {
-    echo "Usage: kubeconfig <user|admin <wc|sc> [cluster_name]>" >&2
+    echo "Usage: kubeconfig user" >&2
```

Comment:
Do we need the `user` argument since it's always user?

Reply:
No you're right, we don't really need it.
Would it be good to keep it to be more backwards compatible or is that just unnecessary?

Reply:
If we can make it so it doesn't error out if you have arguments, then it will be fully compatible with the old way as well.

Reply:
Good point, then I'll change that.